src/Uno.UI/UI/Xaml/Media/Animation/DoubleAnimation.cs

```diff
@@ -462,19 +97,19 @@ protected override void Dispose(bool disposing)
 		{
 			if (disposing)
 			{
-				_subscriptions.Dispose();
-
-				DisposePartial();
+				_animationImplementation.Dispose();
 			}
 
 			base.Dispose(disposing);
 		}
 
-		partial void DisposePartial();
+		float IAnimation<float>.Subtract(float minuend, float subtrahend) => minuend - subtrahend;
+
+		float IAnimation<float>.Add(float first, float second) => first + second;
+
+		float IAnimation<float>.Convert(object value) => Convert.ToSingle(value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5301553)